Data/System/strings.tsv

```diff
@@ -1,61 +1,61 @@
 Settings	Options	Einstellungen	Ajustes	Impostazioni	Inställningar
-Resume game	Reprendre	Spiel fortsetzen	Reanudar el juego	Continua	Återuppta spelet
-Retire game	Abandonner	Spiel abbrechen	Retirarse	Annulla	Dra dig ur spelet
-Quit application	Quitter	Programm beenden	Cerrar el juego	Esci dal gioco	Avsluta programmet
-Save game	Enregistrer la partie	Spiel speichern	Guardar el juego	Risparmi il gioco	Spara spelet
-Continue without saving	Continuer sans enregistrer	Fortfahren	Continuar	Continuano	Fortsätta
-Level	Niveau	Level	Level	Level	Bana
-Score	Score	Score	Score	Score	Poäng
-EMPTY----------	[Vide]	LEER-----------			
+Resume game	Reprendre	Spiel fortsetzen	Reanudar el juego	Riprendi	Återuppta spelet
+Retire game	Abandonner	Spiel abbrechen	Retirarse	Abbandona	Dra dig ur spelet
+Quit application	Quitter	Programm beenden	Cerrar el juego	Esci dal programma	Avsluta programmet
+Save game	Enregistrer la partie	Spiel speichern	Guardar el juego	Salva la partita	Spara spelet
+Continue without saving	Continuer sans enregistrer	Fortfahren	Continuar	Continua senza salvare	Fortsätta
+Level	Niveau	Level	Level	Livello	Bana
+Score	Score	Score	Score	Punteggio	Poäng
+EMPTY----------	[Vide]	LEER-----------		Vuoto	
 Jan.	janv.	Jan	enero	genn.	jan.
-Feb.	févr.	Feb	feb.	febbr.	febr.
+Feb.	févr.	Feb	feb.	febb.	febr.
 Mar.	mars	Mar	marzo	mar.	mars
 Apr.	avr.	Apr	abr.	apr.	april
 May	mai	Mai	mayo	magg.	maj
-June	juin	Jun	jun.	giugno	juni
-July	juil.	Jul	jul.	iuglio	juli
-Aug.	août	Aug	agosto	ag.	aug.
+June	juin	Jun	jun.	giu.	juni
+July	juil.	Jul	jul.	lug.	juli
+Aug.	août	Aug	agosto	ago.	aug.
 Sept.	sept.	Sep	set.	sett.	sept.
 Oct.	oct.	Okt	oct.	ott.	okt.
 Nov.	nov.	Nov	nov.	nov.	nov.
 Dec.	déc.	Dez	dic.	dic.	dec.
-Level Cheat	Choisir un niveau				
-Planet Earth: The Bentley Farm	Planète Terre: La ferme des Bentley				
-Planet Snoth: Area J-3110	Planète Snoth: Zone J-3110				
-Planet Snoth: Slime Boss	Planète Snoth: Le boss visqueux				
-Planet Knarr: Yoth City Ruins	Planète Knarr: Ruines de Yoth City				
-Planet Rennie: Cloud 9	Planète Rennie: Le 7ème ciel				
-Planet Sulak: The South Jungle	Planète Sulak: La jungle méridionale				
-Planet Sulak: Pitcher Plant Boss	Planète Sulak: La plante carnivore				
-Planet Deniz: Valley of Fire and Ice	Planète Deniz: La vallée de feu et de glace				
-Planet Shebanek: POW Camp	Planète Shebanek: Le camp de prisonniers				
-Planet X: The Giant Brain	Planète X: Le cerveau géant				
-Warp to item	Aller à l’objet				
+Level Cheat	Choisir un niveau			Scegli un livello	
+Planet Earth: The Bentley Farm	Planète Terre: La ferme des Bentley			Pianeta Terra: Fattoria di Bentley	
+Planet Snoth: Area J-3110	Planète Snoth: Zone J-3110			Pianeta Snoth: Area J-3110	
+Planet Snoth: Slime Boss	Planète Snoth: Le boss visqueux			Pianeta Snoth: Boss viscido	
+Planet Knarr: Yoth City Ruins	Planète Knarr: Ruines de Yoth City			Pianeta Knarr: Rovine della città di Yoth	
+Planet Rennie: Cloud 9	Planète Rennie: Le 7ème ciel			Pianeta Rennie: Nuvola 9	
```

Comment:
I translated that to "7th sky" in French, which is our loose translation for the English idiom "cloud 9". Feel free to use an Italian sky-related idiom if you guys have one!

Reply:
Oh so, do you think 'Cloud 9' was a reference to this? https://english.stackexchange.com/a/187884